### PR TITLE
Spyder 2.3.7

### PR DESCRIPTION
**Pull request by reitermarkus**

### CODE REVIEW

File: Casks/spyder.rb [hunk 1]
```diff
@@ -1,12 +1,14 @@
 cask :v1 => 'spyder' do
-  version '2.3.6'
-  sha256 'bceb41c323f8910f6e3ad113bd4adbb4d52e996602de146a482b12d57a7bdf82'
+  version '2.3.7'
+  sha256 '014086c5a1ca676efef9de46d97a8fe68f2d7a24a5517c36def429be7692b6fc'
 
   # bitbucket.org is the official download host per the vendor homepage
-  url "https://bitbucket.org/spyder-ide/spyderlib/downloads/spyder-#{version}-py2.7.dmg"
+  url "https://bitbucket.org/spyder-ide/spyderlib/downloads/spyder-#{version}-py3.4.dmg"
+
```
Comment (vitorgalvao) [inline Casks/spyder.rb]: Remove this line. It wasn’t there for a reason.

**Comment by vitorgalvao** [comment]
In the future, please submit separate pull requests for each cask, as more often than not this type of pull request brings problems with one or more of them, and it halts the inclusion of the other ones.

In this case, please submit `spyder-py2` to [caskroom/versions](https://github.com/caskroom/homebrew-versions).

**Comment by reitermarkus** [comment]
Ok, I have removed the extra line and moved `spyder-py2` to https://github.com/caskroom/homebrew-versions.

**Comment by vitorgalvao** [comment]
Thank you.

**Pull request merged**